### PR TITLE
Mh installation fix

### PR DESCRIPTION
**Pull request by MohamadHarastani**
This one has some cleaning and fixes on the installation. It will have conflicts with rv_genesis, I will solve them later and run tests.

### CODE REVIEW

**Comment by MohamadHarastani** [comment]
The major fix is to install the conda environment in scioion/software/em. When the plugin is uninstalled, it deletes the environment. Installing it in this place makes it easy to access its libraries (arepack, etc.)

File: continuousflex/protocols/convert.py [hunk 1]
```diff
@@ -58,9 +59,10 @@ def modeToRow(mode, row):
     
 def getNMAEnviron():
```
Comment (MohamadHarastani) [inline continuousflex/protocols/convert.py:60]: This function adds to the LD_LIBRARY_PATH what is installed on the conda environment.
If you face any issue in running Genesis protocols, you may to use it like its used [here](https://github.com/scipion-em/scipion-em-continuousflex/blob/master/continuousflex/protocols/protocol_nma_base.py#L158)

File: continuousflex/protocols/pdb/protocol_pseudoatoms_base.py [hunk 2]
```diff
@@ -85,7 +86,7 @@ def _insertMaskStep(self, fnVol, prefix=''):
             fnMask = self._getExtraPath('mask%s.vol' % prefix)
             maskParams = '-i %s -o %s --select below %f --substitute binarize'\
                          % (fnVol, fnMask, self.maskThreshold.get())
-            self._insertRunJobStep('xmipp_transform_threshold', maskParams)
+            self.runJob("xmipp_transform_threshold", maskParams, env=Domain.importFromPlugin('xmipp3').Plugin.getEnviron())
```
Comment (MohamadHarastani) [inline continuousflex/protocols/pdb/protocol_pseudoatoms_base.py:89]: Such changes allowed to remove xmipp from getEnviron of the __init__.py

File: continuousflex/protocols/pdb/protocol_pseudoatoms_base.py [hunk 3]
```diff
@@ -106,17 +107,17 @@ def convertToPseudoAtomsStep(self, inputFn, fnMask, sampling, prefix=''):
                   "-v 2 --intensityColumn Bfactor"
         if fnMask:
             params += " --mask binary_file %(fnMask)s"
-        self.runJob("xmipp_volume_to_pseudoatoms", params % locals())
+        self.runJob("xmipp_volume_to_pseudoatoms", params % locals(), env=Domain.importFromPlugin('xmipp3').Plugin.getEnviron())
```
Comment (mms29) [inline continuousflex/protocols/pdb/protocol_pseudoatoms_base.py:110]: Do we need to add this now each time we call a Xmipp program ?

**Review by mms29 — APPROVED** [review]
I installed and ran all tests, everything is fine, great job!

File: continuousflex/protocols/pdb/protocol_pseudoatoms_base.py (hunk 3 above)
Comment (MohamadHarastani) [reply continuousflex/protocols/pdb/protocol_pseudoatoms_base.py:110]: Actually it depends how you call the program. If you use self.runJob() then yes. There is a function that can be used without passing this environment called runProgram() 
https://github.com/scipion-em/scipion-em-continuousflex/blob/master/continuousflex/protocols/protocol_tomoflow.py#L36
However, when there is an mpi job (which goes and uses the binary that starts with mpi_) , runJob does it automatically. Here I could use runProgram

**Pull request merged**